- Minor typo fix.

antoniorozsos opened the pull request:
[...][ The implementation is provided by an method [override][...] 
- Added white space between the word 'method' and the keyword 'override'.

# Title

On the title describe
what you've fixed (or created) with this Pull Request (PR).

## Summary

Insert a short (one or two sentence) summary here.

Fixes #Issue_Number

>Note: The "Fixes #nnn" syntax in the PR description causes
>GitHub to automatically close the issue when this PR is merged.
> Remove that line if you don't have issues associated with this
> PR. Click on the Guidelines for Contributing link above for details.

## Details

Explain your changes, and why you made them. If that
information is already available in the issue referenced
above, just referencing the issue is preferred to copying
the text.

This may not be necessary depending on the scope of the PR 
changes. For example, "fix typo in introduction.md" is
sufficient to describe that PR.

## Suggested Reviewers

If you know who should review this, use '@' to request a review.

antoniorozsos committed:
Typo fix

[...][ The implementation is provided by an overriding method [override][...] 
- Added white space between the word 'method' and the keyword override.

rpetrusha reviewed (changes requested):
Thanks for contributing to the dotnet/docs repo, @antoniorozsos, and for correcting this typo. It occurs to me, though, that the repetition of "override" in "overriding method override" is a very confusing. Would you mind deleting "overriding" so that it reads "provided by a method \[override]"?

antoniorozsos committed:
Update abstract.md

- Removed the word "overriding" from the sentence "The implementation is provided by an overriding method [override]" so it is less confusing for readers.
- The sentence will now be "The implementation is provided by an method [override]"

antoniorozsos commented:
Hello @rpetrusha, modified as requested. Please take a look ^^.

rpetrusha reviewed (approved):
Thank you so much, @antoniorozsos, for making the additional change. We'll merge your PR now. Your changes should appear live on docs.microsoft.com in the next day or two.
